gtk/src/auto/container.rs

```diff
@@ -175,6 +186,19 @@ impl<O: IsA<Container>> ContainerExt for O {
     //    unsafe { TODO: call ffi:gtk_container_child_get() }
     //}
 
+    fn child_get_property(&self, child: &impl IsA<Widget>, property_name: &str) -> glib::Value {
+        unsafe {
+            let mut value = glib::Value::uninitialized();
+            ffi::gtk_container_child_get_property(
```

Comment:
This needs checking if the property exists and is readable. You return garbage from here if that's not the case :)

Reply:
This basically needs something like there's in ObjectExt